addons/website_event/views/event_templates_list.xml

```diff
@@ -289,96 +305,166 @@
+                                    <div t-if="not event.address_id or has_city_or_country" class="d-flex align-items-center">
+                                        <i t-attf-class="fa fa-map-marker me-2 #{(not event.event_registrations_open) and 'text-muted'}" title="Location"/>
+                                        <small t-if="event.address_id" t-attf-class="o_not_editable fw-bold #{(not event.event_registrations_open) and 'text-muted'}" itemprop="location" t-out="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'no_marker': 'true'}"/>
+                                        <small t-else="" t-attf-class="o_not_editable fw-bold #{(not event.event_registrations_open) and 'text-muted'}" itemprop="location">Online event</small>
+                                    </div>
+                                    <!-- Tags -->
+                                    <div class="d-flex flex-wrap  gap-1 small">
+                                        <t t-foreach="event.tag_ids.filtered(lambda tag: tag.category_id.website_id == website or not tag.category_id.website_id)" t-as="tag">
+                                            <span t-if="tag.color"
+                                                t-attf-class="badge rounded-pill d-flex align-items-center px-2 p-1 #{'o_badge_color_%s' % tag.color if tag.color else 'text-bg-light'}">
+                                                <t t-out="tag.name"/>
+                                            </span>
+                                        </t>
+                                    </div>
                                 </div>
-                                <!-- Not open -->
-                                <span t-if="not event.event_registrations_open and (not opt_events_list_cards or not opt_events_list_columns)" class="position-absolute bottom-0 px-3 py-2 w-100 text-bg-light">
-                                    <t t-if="not event.event_registrations_started">
-                                        Registrations not yet open
+                            </div>
+                            <!-- Action Button -->
+                            <div t-attf-class="d-none #{opt_index_sidebar and 'd-xl-block' or 'd-lg-block'}">
+                                <div class="d-flex flex-column align-items-end gap-1">
+                                    <t t-if="not event.event_registrations_open">
+                                        <small class="text-muted fw-bold">
+                                            <t t-if="not event.event_registrations_started and not event.event_registrations_sold_out">
+                                                Registrations not open
+                                            </t>
+                                            <t t-elif="event.event_registrations_sold_out">
+                                                Sold Out
+                                            </t>
```

Comment:
So we want to show sold out before we show registrations not open? Makes sense indeed